### ISSUE TITLE
[BUG] Pickling error on use of ReLU

### ISSUE THREAD

**Issue by MilesCranmer** [MilesCranmer/PySR#245]
I see this error when I try to use the ReLU operator:
```
PicklingError: Can't pickle relu: attribute lookup relu on __main__ failed
```

seems like it's implemented in a way that can't be pickled. Should be an easy fix.

**Comment by MilesCranmer** [comment]
Looks like `max` and `min` also aren't implemented in the default dictionaries?

**Comment by MilesCranmer** [comment]
Sorry this took me so long to get around to. Fixed with #486
